README.md

```diff
@@ -28,6 +28,17 @@ The latest release of HwameiStor is [![hwameistor-releases](https://img.shields.
 
 See [current releases](https://github.com/hwameistor/hwameistor/releases).
 
+## Running Environments
+
+### Kubernetes compatiblity
+| kubernetes | v0.4.3     | 
+| ---------- | ---------- |
+| >=1.18&&<=1.20 | support    | 
+| 1.21       | support    | 
+| 1.22       | support    | 
+| 1.23       | support    | 
+| 1.24       | support    | 
+| 1.25       | nonsupport | 
```

Comment:
Please also update the "Release Status" section with the latest release version.